### ISSUE TITLE
PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/pdo.so'

### ISSUE THREAD

**Issue by nadar** [shivammathur/setup-php#388]
**Describe the bug**

Since today (maybe even earlier but it was working at least 7 days ago) we receive the following error:

```
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/pdo.so' - /usr/lib/php/20151012/pdo.so: cannot open shared object file: No such file or directory in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/pdo_mysql.so' - /usr/lib/php/20151012/pdo_mysql.so: undefined symbol: pdo_parse_params in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/pdo_pgsql.so' - /usr/lib/php/20151012/pdo_pgsql.so: undefined symbol: pdo_parse_params in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/pdo_sqlite.so' - /usr/lib/php/20151012/pdo_sqlite.so: undefined symbol: php_pdo_unregister_driver in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/pdo_sqlite.so' - /usr/lib/php/20151012/pdo_sqlite.so: undefined symbol: php_pdo_unregister_driver in Unknown on line 0
```

See the action: https://github.com/luyadev/luya-module-payment/runs/1648794919. Maybe its related to https://github.com/shivammathur/setup-php/issues/387 but i am not sure about.

**Version**
- [ ] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

Update: I have tested with `uses: shivammathur/setup-php@master` and the problem still persists.

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**

`os: [ubuntu-latest]`

**PHP versions**

Everything works from 7.1 - 7.4 but **not** with 7.0

**To Reproduce**

https://github.com/luyadev/luya-module-payment/blob/master/.github/workflows/tests.yml

**Expected behavior**

It should work like 7 days ago :-) see https://github.com/luyadev/luya-module-payment/actions/runs/450889409 Nothing has changed in the workflow yaml since then

**Are you willing to submit a PR?**

sure, if i can help i would :-)

**Comment by shivammathur** [comment]
Fixed in 7c8e8da

**Comment by nadar** [comment]
Works again, thanks for fixing the problem that quick! https://github.com/luyadev/luya-module-payment/runs/1649632048
